CHANGELOG.md

```diff
@@ -1,3 +1,47 @@
+## [Unreleased]
+<details>
+  <summary>
+    Changes that have landed in master but are not yet released.
+    Click to see more.
+  </summary>
+
+### JS Environment Requirements
```

Comment:
Maybe worth changing to "New JS Environment Requirements" to better signal it is a change.